.changeset/thirty-owls-lay.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/core': patch
+'storybook': patch
+---
+
+exposes undocumented PageTheme
```

Comment:
Make `PageTheme` an inline code block, that way it doesn't need to be added to vocab

Reply:
done